### ISSUE TITLE
how to handle callback / post-render behaviors ?

### ISSUE THREAD

**Issue by timelyportfolio** [ramnathv/htmlwidgets#65]
Do we expect `htmlwidgets` to handle `callback` or other post-render behaviors, or should we expect widget authors to handle on their end?  `dygraphs` handles very nicely.  I wonder though given potential lack of Javascript knowledge on the part of some widget authors if we can establish some best practices for this.  See https://github.com/rich-iannone/DiagrammeR/pull/10.

**Comment by jjallaire** [comment]
I think widget authors need to handle this. We've given them the JS
function and some examples of it's use, hopefully that (and pull requests
from us!) will be enough to get people pointed in the right direction.

**Comment by timelyportfolio** [comment]
I agree for now, but the one situation where dealing with this at the individual widget level might be problematic will be a page with multiple `htmlwidgets` from multiple packages where the package authors differ in their implementation or do not implement.  Then, finding when all widgets are built will be troublesome.
